.github/workflows/podman-desktop-e2e-windows.yaml

```diff
@@ -0,0 +1,158 @@
+            --windows-version '${{ matrix.windows-version }}' \
+            --windows-featurepack '${{ matrix.windows-featurepack }}' \
+            --vmsize 'Standard_D8a_v4' \
+            --tags project=podman-desktop \
+            --spot
+        # Check logs 
+        podman logs -f windows-create
+
+    - name: Check instance system info
+      run: |
+        ssh -i id_rsa \
+          -o StrictHostKeyChecking=no \
+          -o UserKnownHostsFile=/dev/null \
+          -o ServerAliveInterval=30 \
+          -o ServerAliveCountMax=1200 \
+          $(cat username)@$(cat host) "systeminfo"
+
+    - name: Emulate X session 
+      run: |
+        # use fake rdp to emulate an active x session
+        podman run -d --name x-session \
+          -e RDP_HOST=$(cat host) \
+          -e RDP_USER=$(cat username) \
+          -e RDP_PASSWORD=$(cat userpassword) \
+          quay.io/rhqp/frdp:v0.0.1
+        # Wait until the x session has been created
+        podman wait --condition running x-session
+        # Check logs for the x session
+        podman logs x-session
+
+    - name: Run PDE2E builder and setup image
+      run: |
+        podman run -d --name pde2e-builder-run \
+          -e TARGET_HOST=$(cat host) \
+          -e TARGET_HOST_USERNAME=$(cat username) \
+          -e TARGET_HOST_KEY_PATH=/data/id_rsa \
+          -e TARGET_FOLDER=pd-e2e \
+          -e TARGET_RESULTS=results \
+          -e OUTPUT_FOLDER=/data \
+          -e DEBUG=true \
+          -v $PWD:/data:z \
+          quay.io/odockal/pde2e-builder:v0.0.1-snapshot  \
+              pd-e2e/run.ps1 \
+                  -targetFolder pd-e2e \
+                  -resultsFolder results \
+                  -fork ${{ inputs.fork }} \
+                  -branch ${{ inputs.branch }}
+        # check logs
+        podman logs -f pde2e-builder-run
+        
+    - name: Run podman desktop e2e
+      run: |
+        # Get latest built 
+        # tag=$(curl --silent https://api.github.com/repos/containers/podman-desktop/releases | jq -r 'map(select(.prerelease)) | first | .tag_name')
+        pwd
+        ls
+        cat results/pde2e-builder-results.log
+        pdPath=$(cat results/pde2e-builder-results.log)
+        # Run e2e tests
+        podman run --rm -d --name pd-e2e-windows \
+          -e TARGET_HOST=$(cat host) \
+          -e TARGET_HOST_USERNAME=$(cat username) \
+          -e TARGET_HOST_KEY_PATH=/data/id_rsa \
+          -e TARGET_FOLDER=pd-e2e \
+          -e TARGET_RESULTS=pd-e2e-results.xml \
+          -e OUTPUT_FOLDER=/data \
+          -e DEBUG=true \
+          -v $PWD:/data:z \
+          quay.io/rhqp/podman-desktop-e2e:v1.1.0-windows-amd64  \
+              pd-e2e/run.ps1 \
+                  -targetFolder pd-e2e \
+                  -pdPath $pdPath \
+                  -junitResultsFilename pd-e2e-results.xml 
+        # Check logs 
+        podman logs -f pd-e2e-windows
+
+    - name: Run PDE2E Runner image
```

Comment:
I guess this should be implemented here https://github.com/odockal/pde2e-runner but I see no code there, can you pushed to see what is doing?

Reply:
thx. fixed.